### ISSUE TITLE
Titles in MeSH controller not showing up on first load

### ISSUE THREAD

**Issue by adjavaherian** [adjavaherian/EMA2#3]
Mesh tab -> mesh descriptor -> titles not visible on initial load.

Looks like the cells are not refreshing.

**Comment by adjavaherian** [comment]
MeSH tabs are showing up, but super slowly.  https://youtu.be/P1AhrOYBPOA

**Comment by adjavaherian** [comment]
I think this was due to not having added these indexes in previous db builds.  should be fixed in 1.0.7 and higher

```sql
CREATE INDEX abstract_mesh_pmid_mesh_id_idx ON abstract_mesh(pmid, mesh_id);
CREATE INDEX abstract_mesh_meshid_pmid ON abstract_mesh(mesh_id, pmid);
CREATE INDEX e_pmid ON erpubtbl(pmid);
```
